src/sentry/api/endpoints/release_thresholds/release_threshold_details.py

```diff
@@ -43,6 +43,7 @@ def validate_trigger_type(self, trigger_type: str):
 
 @region_silo_endpoint
 class ReleaseThresholdDetailsEndpoint(ProjectEndpoint):
+    permission_classes = (ProjectAlertRulePermission,)
```

Comment:
I checked this permission and it includes `alerts:read/write` but not `project:releases`. Does that make sense as I thought this would mirror the existing release endpoint permissions

Reply:
Ah thank you
yes - i'll switch to that